runtime/src/bank.rs

```diff
@@ -4126,10 +4126,13 @@ impl Bank {
 
     pub fn check_transaction_for_nonce(
         &self,
-        _tx: &SanitizedTransaction,
+        tx: &SanitizedTransaction,
     ) -> Option<TransactionAccount> {
-        None
-        // self.check_message_for_nonce(tx.message())
+        if self.slot() > 135986379 {
```

Comment:
Can we wrap the entire logic with a `self.cluster_type() == ClusterType::MainnetBeta`

Reply:
Yes plz

Reply:
done